powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/EncryptionServiceBehavior.java

```diff
@@ -67,6 +71,11 @@ public void setLocalizationProvider(LocalizationProvider localizationProvider) {
         this.localizationProvider = localizationProvider;
     }
 
+    @Autowired
```

Comment:
dtto - using constructor injection?

Reply:
I chose setter injection in ActivationServiceBehavior, so I followed the same pattern in all classes.